### ISSUE TITLE
LocalStorage data format

### ISSUE THREAD

**Issue by RubenVerborgh** [inrupt/solid-auth-fetcher#105]
Does solid-auth-fetcher also storage data in LocalStorage?

If so, what does that data look like?

I'm asking because we had some design flaws in the storage format of solid-auth-client that made upgrading difficult, so I'd want to verify that we've made better choices here.

**Comment by jaxoncreed** [comment]
Yep it stores things in local storage. Here's what it looks like:

<img width="930" alt="image" src="https://user-images.githubusercontent.com/5464743/82477641-becefa80-9a9d-11ea-9743-492e25887d64.png">

For the most part it just ignores what was stored for solid-auth-client. So user's will need to log in again after an upgrade.

**Comment by RubenVerborgh** [comment]
I think we want to version this information somehow (which we didn't in solid-auth-client). I'd suggest the version number of the solid-auth-fetcher that wrote it; (only) major version numbers can break it.
